### DISCUSSION TITLE
Error applying for student pack

### DISCUSSION THREAD

**Discussion by arentalb** [community/community#113809] [GitHub Education]
Hello , i have contacted Github before but the problem i had didnot fixed , my problem is that in the video , I upload my document but it dose not give me any response if my application submitted or failed and i really need that student pack and i dont know how fix that , i did what is did in the video more than 10 times from different devices and different browsers , thanks

https://github.com/community/community/assets/70669302/1df03a0c-57cc-49d2-ab7c-2b9ad7f67c44

**Comment by Jaishree-04** [comment]
I have a problem in GitHub educational pack in which my official documents have the last name in single letter but GitHub does not allow it. Do you know what to do? Kindly help I need it definitely

**Comment by jasonmiller2332** [comment]
It sounds like the submission isn't completing at all. Since you've already tried different browsers and devices, it may be an issue with GitHub's verification system rather than your setup. I'd recommend replying to your existing support ticket, attaching the video, and mentioning that the problem persists across multiple devices and networks. Hopefully they can manually review your application or identify what's blocking the submission. Good luck!

**Comment by benogoulart** [comment]
Hi,

It sounds like your application isn't being submitted successfully if you don't receive any confirmation or rejection after uploading your documents.

Since you've already tried multiple browsers and devices, there are a couple of additional things you can check:

- Try submitting from a different network (for example, a mobile hotspot instead of your home network).
- Make sure all required fields are completed before uploading your document.
- Check your browser's Developer Tools (**F12 → Console/Network**) for any errors that occur when you click **Submit**.
- Verify that your document meets the supported file type and size requirements.

If the issue persists and the application never reaches the submission stage, it's likely a problem with the application flow rather than your documentation.

For **@Jaishree-04**, if your official documents only include a single-letter last name, I'd recommend opening a new discussion specific to your situation, as it's a different issue from the original post and may require guidance specific to GitHub Education's name verification process.

I hope you're both able to get your applications submitted successfully.

**Comment by ummeaymen499** [comment]
My GitHub Education application has now been rejected twice because GitHub could not verify that I was near my selected educational institution. I had already selected the appropriate option during my application indicating that my semester has not started yet. I am currently in my hometown during my summer break, so I am temporarily away from my university campus. Once the new semester begins, I will return to my university in Lahore and stay in the hostel as usual. I would also like to mention that I have previously been a GitHub Student Developer Pack member and successfully used its benefits for the past two years. I recently received an email asking me to verify my academic status again, which is why I submitted a new application.

**Comment by nishec** [comment]
It might be the case that your ID Card does not have sufficient information for them to verify. Try uploading Examination Admit Card, or some Marksheet. That should prolly work.
